### PR TITLE
readonlymind adapter

### PR DESCRIPTION
**Pull request by Nothorse**
New adapter for readonlymind.com

(Hope I did it right)

### CODE REVIEW

File: fanficfare/adapters/adapter_readonlymindcom.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        # get storyId from url
+        self.story.setMetadata('storyId',self.parsedUrl.path.split('/',)[2])
+
+
+        # The date format will vary from site to site.
+        # http://docs.python.org/library/datetime.html#strftime-strptime-behavior
+        self.dateformat = "%Y-%m-%d"
+
+    ################################################################################################
+    def getBaseDomain(self):
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_readonlymindcom.py:72]: getBaseDomain() isn't needed.

File: fanficfare/adapters/adapter_readonlymindcom.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    ################################################################################################
+    @classmethod
+    def getAcceptDomains(cls):
+
+        return ['readonlymind.com', 'www.readonlymind.com']
+
+    ################################################################################################
+    @classmethod
+    def getSiteExampleURLs(self):
+        return ("https://readonlymind.com/@AnAuthor/A_Story_Name/ ")
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_readonlymindcom.py:91]: Return string, not tuple please.

File: fanficfare/defaults.ini [hunk 3]
```diff
@@ -2323,6 +2323,20 @@ add_to_comma_entries:,pages,readers,reads,favorites,comments
 
 website_encodings: utf8:ignore, Windows-1252, iso-8859-1
 
+[readonlymind.com]
```
Comment (JimmXinu) [inline fanficfare/defaults.ini:2326]: Should also be added to `calibre-plugin/plugin-defaults.ini`.

File: fanficfare/adapters/adapter_readonlymindcom.py [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+            self.add_chapter(chapterTitle, chapterUrl)
+
+
+        logger.debug("Story: <%s>", self.story)
+
+        return
+
+    def getChapterText(self, url):
+        """
+        Clean up a mcstories chapter page.
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_readonlymindcom.py:174]: Remove or correct references to mcstories please.

File: fanficfare/adapters/adapter_readonlymindcom.py [hunk 5]
```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# -- coding: utf-8 --
+# Copyright 2013 Fanficdownloader team, 2021 FanFicFare team
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_readonlymindcom.py:3]: Is there a reason for the two different coding comments?
Copyright on a new file should only be this year.

**Review by JimmXinu — CHANGES_REQUESTED** [review]
Some nit picks.

File: fanficfare/adapters/adapter_readonlymindcom.py (hunk 1 above)
Comment (Nothorse) [reply fanficfare/adapters/adapter_readonlymindcom.py:72]: OK

File: fanficfare/defaults.ini (hunk 3 above)
Comment (Nothorse) [reply fanficfare/defaults.ini:2326]: Ah, didn't see that ini, as I don't use calibre. Sure.

File: fanficfare/adapters/adapter_readonlymindcom.py (hunk 2 above)
Comment (Nothorse) [reply fanficfare/adapters/adapter_readonlymindcom.py:91]: OK.

File: fanficfare/adapters/adapter_readonlymindcom.py (hunk 4 above)
Comment (Nothorse) [reply fanficfare/adapters/adapter_readonlymindcom.py:174]: OOps. :)

File: fanficfare/adapters/adapter_readonlymindcom.py (hunk 5 above)
Comment (Nothorse) [reply fanficfare/adapters/adapter_readonlymindcom.py:3]: That's just what was in the mcstories adapter. I can remove the old one.

**Comment by Nothorse** [comment]
Fixed it all, I think

File: fanficfare/adapters/adapter_readonlymindcom.py [hunk 6]
```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-# -- coding: utf-8 --
-# Copyright 2013 Fanficdownloader team, 2021 FanFicFare team
+# Copyright 2021 FanFicFare team
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_readonlymindcom.py:2]: Nittest of nit picks--it's 2022 now. :-)

**Review by JimmXinu — CHANGES_REQUESTED** [review]

File: fanficfare/adapters/adapter_readonlymindcom.py (hunk 6 above)
Comment (Nothorse) [reply fanficfare/adapters/adapter_readonlymindcom.py]: :D 

January, you know.

**Review by JimmXinu — APPROVED** [review]

**Pull request merged**